bindings/julia/Displaz.jl

```diff
@@ -50,15 +49,54 @@ function write_ply_points(fileName, nvertices, fields)
             for i = 1:size(value,2)
                 propName = ply_property_name(semantic, i)
                 write(fid, "property $typeName $propName\n")
-            end
+        	end
         end
         write(fid, "end_header\n")
         for (_,value) in convertedFields
-            write(fid, value')
+			write(fid, value')
         end
     end
 end
 
+function write_ply_lines(fileName, position, color)
+	vertexValid = ~all(isnan(position),2)
```

Comment:
It's a bit unfortunate to inherit the matlab hack of separating the position array using nans.  (The only reason the matlab bindings do it is because it's a relatively fast way to plot a lot of disconnected lines using the builtin matlab plot command.)

Ideally I'd like to settle on a more sensible convention (for instance, passing the vector of lineStarts explicitly)